src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -869,14 +875,14 @@ export function findAttributeInDimension(
     dimension: any,
     attributeHeaderItemsDimension: any,
     indexInDimension?: number
-) {
+): IUnwrappedAttributeHeadersWithItems {
     return findInDimensionHeaders(
         [dimension],
         (headerType: any, header: any, _dimensionIndex: number, headerIndex: number) => {
             if (headerType === 'attributeHeader'
                 && (indexInDimension === undefined || indexInDimension === headerIndex)) {
                 return {
-                    ...header,
+                    ...header, // TODO Do not unwrap header here since we have to wrap header again for drilling
```

Comment:
`TODO` ?

Reply:
I'm sorry, but feractoring this means add types and refactoring half of this module. I cannot afford this right now. I will remove the comment and write some notes for possible followup.